contributingGuides/TS_STYLE.md

```diff
@@ -28,6 +28,7 @@
   - [1.21 `compose` usage](#compose-usage)
   - [1.22 Type imports](#type-imports)
   - [1.23 Ref types](#ref-types)
+  - [1.24 Consice Way To Access or Default](#concise-way)
```

Comment:
Can you try rewording this so that the meaning is more clear? I'm not really sure what it's trying to say.